src/compiler/config/outputs/validate-angular.ts

```diff
@@ -1,7 +1,8 @@
-import type * as d from '../../../declarations';
-import { isOutputTargetAngular } from '../../output-targets/output-utils';
 import { isAbsolute, join } from 'path';
+
+import type * as d from '../../../declarations';
 import { OutputTargetAngular } from '../../../declarations';
```

Comment:
Heh - I wouldn't change this (since I removed this file in the `v3.0.0-dev` branch). Just funny